172.md

```diff
@@ -0,0 +1,94 @@
+  "created_at": "<Unix timestamp in seconds>",
+  "kind": "34550",
+  "tags": [
+    ["d", "<community_name>"],
+    ["description", "<community_description>"],
+	["image",  "<communityimagen>", "WidthxHeight"]
+	
+	//.. other tags relevant to defining the community
+
+    // moderators
+    ["p", "<32-bytes hex of a pubkey>", "<optional recommended relay URL>", "moderator"],
+    ["p", "<32-bytes hex of a pubkey>", "<optional recommended relay URL>", "moderator"],
+    ["p", "<32-bytes hex of a pubkey>", "<optional recommended relay URL>", "moderator"],
+	
+	// relays used by the community 
+    ["relay", "<relay hosting author kind 0>", "author"],
+    ["relay", "<relay where to post requests to and fetch approvals from>"], 
+    ["relay", "<relay where to post requests to and fetch approvals from>"]
+  ]
+}
+```
+
+# New Post Request
+
+Any Nostr event can be a post request. Clients should simply add the community's `a` tag to be presented for the moderator's approval.
+
+```js
+{
+  "id": "<32-bytes lowercase hex-encoded SHA-256 of the the serialized event data>",
+  "pubkey": "<32-bytes lowercase hex-encoded public key of the event creator>",
+  "created_at": "<Unix timestamp in seconds>",
+  "kind": "1",
+  "tags": [
+    ["a", "34550:<community event author pubkey>:<d-identifier of the community>", "<optional relay url>"],
+  ], 
+  "content": "<my content>"
+}
+```
+
+Community management clients can filter all mentions of the kind-34550 event and request moderators to approve each submission. The same moderator can remove his/her approval of the post at any time.
+
+# Post Approval by moderators
+
+The post-approval event includes a stringified `new post request` event inside the `.content` of the approval (NIP-18-style).
+
+```js
+{
+  "id": "<32-bytes lowercase hex-encoded SHA-256 of the the serialized event data>",
+  "pubkey": "<32-bytes lowercase hex-encoded public key of the event creator>",
+  "created_at": "<Unix timestamp in seconds>",
+  "kind": "34551",
+  "tags": [
+    ["a", "34550:<community event author pubkey>:<d-identifier of the community>", "<optional relay url>"],
+    ["e", "<Post Request ID>", "<optional relay url>"],
+    ["p", "<Post Request Author ID>", "<optional relay url>"],
+    ["k", "<New Post Request kind>"],
+  ], 
+  "content": "{ <New Post Request JSON> }"
+}
+```
+
+It's recommended that multiple moderators approve posts to avoid unapproving them when a given moderator is removed from the owner's list. In case the full list of moderators must be rotated, the new moderator set must sign post-approvals for posts in the past or the community will restart.
+
+# Displaying
+
+Community clients can display posts that have been approved by at least 1 moderator or by the community owner.
+
+The following filter displays the approved posts.
+
+```js
+{
+  "authors": ["<author>", "moderator1", "moderator2", "moderator3", ...],
+  "kinds": 34551,
```

Comment:
This should be an array. Maybe the event kind should be a different number (see comment above)

Reply:
Fixed